R/gcalendr_auth.R

```diff
@@ -95,23 +92,18 @@ calendar_auth <- function(
   invisible()
 }
 
-#' Suspend authorization
+#' Clear current token
```

Comment:
I don't think you really have a "de-authed" mode with this API. So this function just clears the token. bigrquery is same.